openassessment/staffgrader/staff_grader_mixin.py

```diff
@@ -132,6 +132,31 @@ def delete_submission_lock(self, submission_uuid, data, suffix=''):  # pylint: d
         except SubmissionLockContestedError as err:
             raise JsonHandlerError(403, err.get_error_code()) from err
 
+    @XBlock.json_handler
+    @require_course_staff("STUDENT_GRADE")
+    def batch_delete_submission_lock(self, data, suffix=''):  # pylint: disable=unused-argument
+        """
+        Given a list of submission UUIDs, clear those that we currently have locks for.
+
+        Returns: None, no errors is implicit success
+
+        Raises:
+        - 400 in the case of bad params/data
+        - 500 for generic errors
+        """
+        anonymous_user_id = self.get_anonymous_user_id_from_xmodule_runtime()
```

Comment:
Stupid question: is there a reason we don't want this class of error to be louder by default (i.e. no runtime or no anon ID = exceptions)? Feels like this is unnecessarily lenient and would be more likely to lead to errors like you're talking about.

For this case, happy to add that error checking but it seems like an annoying one-off since we aren't doing that in other places. I tend to like to follow the patterns established in the code so we have to keep fewer of these different cases in our heads.